config/resources/government.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "kans": "http://kanselarij.vo.data.gift/core/",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "ext": "http://mu.semte.ch/vocabularies/ext/"
+  },
+  "resources": {
+    "government-fields": {
+      "new-resource-base": "http://themis.vlaanderen.be/beleidsveld/",
+      "class": "kans:Beleidsveld",
+      "attributes": {
+        "prefLabel": {
+          "type": "string",
+          "predicate": "skos:prefLabel"
+        }
+      },
+      "relationships": {
+        "hasGovernmentDomain": {
```

Comment:
De attribute/relatie-naam hoeft geen exacte vertaling van het predicaat te zijn. De naam van het attribuut/relatie wordt vooral gekozen vanuit het oogpunt van een developer, terwijl het predicaat een semantische betekenis omvat. Doorgaans gebruiken we voor een relatie gewoon de naam van het target type, tenzij er meerdere soorten relaties naar hetzelfde target type zijn. 

Dat zou hier dan `government-domain` zijn ipv `has-government-domain`. Op die manier kan dit in de frontend gebruikt worden als `governmentField.governmentDomain` wat natuurlijker leest dan `governmentField.hasGovernmentDomain`. Bij dat laatste zou je in de frontend code eerder een boolean verwachten dan een `government-domain` resource.